src/main/scala/dotty/dokka/model/api/api.scala

```diff
@@ -98,10 +98,16 @@ extension (s: Signature):
   def join(a: Signature): Signature = s ++ a
 
 case class LinkToType(signature: Signature, dri: DRI, kind: Kind)
-
-case class HierarchyDiagram(edges: Seq[Edge])
-case class Vertex(val id: Int, val body: LinkToType)
-case class Edge(val from: Vertex, val to: Vertex)
+case class HierarchyGraph(edges: Seq[(LinkToType, LinkToType)]):
+  def vertecies: Seq[LinkToType] = edges.flatten((a, b) => Seq(a, b)).distinct
```

Comment:
Usually `flatMap` is used in such cases, I am supprised that `flatten` works

Reply:
Would you like me to change it then?

Reply:
we can leave it for now